UnicornRender/video/include/unicorn/video/CameraFpsController.hpp

```diff
@@ -20,6 +20,8 @@ class CameraFpsController : public CameraController
 {
 public:
     UNICORN_EXPORT CameraFpsController(glm::mat4& cameraView);
+    UNICORN_EXPORT void RotateLeft(float degrees);
```

Comment:
I think rotations should be described as CW and CCW (clockwise and counterclockwise).

Also for FPS control I think it makes sense to add a pivot point as well as angle. So that user doesn't have to do all calculations themselves. By default pivot point can be origin.